atomic_reactor/plugins/pre_fetch_sources.py

```diff
@@ -271,10 +267,10 @@ def _get_kojifile_source_urls_helper(self, koji_build):
             source_build = self.session.getBuild(build_id, strict=True)
             if source_build['owner_name'] == PNC_SYSTEM_USER:
                 pnc_build_id = source_build['extra']['external_build_id']
-                url, dest_filename = pnc.get_scm_archive_from_build_id(
-                                                                    api_request_url=api_request_url,
-                                                                    build_id=pnc_build_id,
-                                                                    session=req_session)
+                if not self.pnc_util:
+                    self.pnc_util = PNCUtil(self.workflow)
```

Comment:
Would be safer to make `pnc_util` lazy property?

Reply:
Updated to make it a lazy_property